Packs/Iboss/Integrations/Iboss/iboss.yml

```diff
@@ -0,0 +1,751 @@
+      required: false
+      secret: false
+      defaultValue: '0'
+      predefined:
+      - '0'
+      - '1'
+      auto: PREDEFINED
+    - name: direction
+      isArray: false
+      description: Which direction(s) to match.
+      required: false
+      secret: false
+      defaultValue: '2'
+      predefined:
+      - '0'
+      - '1'
+      - '2'
+      auto: PREDEFINED
+    - name: start_port
+      isArray: false
+      description: Which start port(s) to match; 0 indicates all ports.
+      required: false
+      secret: false
+      defaultValue: '0'
+    - name: end_port
+      isArray: false
+      description: Which end port(s) to match; 0 indicates all ports.
+      required: false
+      secret: false
+      defaultValue: '0'
+    - name: global
+      isArray: false
+      description: Whether to apply to all groups.
+      required: false
+      secret: false
+      defaultValue: '0'
+      predefined:
+      - '0'
+      - '1'
+      auto: PREDEFINED
+    - name: is_regex
+      isArray: false
+      description: Whether entity consists of a regex pattern.
+      required: false
+      secret: false
+      defaultValue: '0'
+      predefined:
+      - '0'
+      - '1'
+      auto: PREDEFINED
+    - name: priority
+      isArray: false
+      description: Priority of entry (higher number takes precedence) when conflicting entry in the block list.
+      required: false
+      secret: false
+      defaultValue: '0'
+    - name: time_url_expires_in_minutes
+      isArray: false
+      description: The expiration time in minutes for the entry (0 indicates an entry that does not expire).
+      required: false
+      secret: false
+      defaultValue: '0'
+    - name: note
+      isArray: false
+      description: Note added to the entry.
+      required: false
+      secret: false
+    outputs:
+    - contextPath: iboss.AddEntityToBlockList.message
+      description: Operation result.
+      type: String
+  - deprecated: false
+    description: Lookup reputation for domain names.
+    name: domain
+    arguments:
+    - name: domain
+      isArray: true
+      description: Domain(s) to lookup.
+      required: true
+      secret: false
+      default: true
+    outputs:
+    - contextPath: DBotScore.Indicator
+      description: The indicator.
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The DBot score.
+      type: Number
+    - contextPath: Domain.Malicious.Description
+      description: The indicator context description
+      type: String
+    - contextPath: Domain.Malicious.Vendor
+      description: The vendor that indicator context originated from.
+      type: String
+    - contextPath: Domain.Name
+      description: The domain.
+      type: String
+    - contextPath: iboss.isSafeUrl
```

Comment:
I can see that in the command you set the context output prefix to:
f'{CONTEXT_PREFIX}.Domain'
That means you'll need to change the contextPaths here in the YML to: 
iboss.Domain. ...

For example use 'iboss.Domain.isSafeUrl' instead of: iboss.isSafeUrl .

When you'll finish updating the YML file with those changes please generate the readme file again using the: `demisto-sdk generate-docs` command (with the -e flag) - for mire info see [this](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md).

Reply:
Updated